ecosystem/sep-0024.md

```diff
@@ -549,6 +549,7 @@ Name | Type | Description
 `kind` | string | `deposit` or `withdrawal`.
 `status` | string | Processing status of deposit/withdrawal.
 `status_eta` | number | (optional) Estimated number of seconds until a status change is expected.
+`kyc_verified` | boolean | (optional) Whether or not the user's KYC information has been verified by the anchor.
```

Comment:
Should we add a qualifications here that KYC has been verified for this specific transaction? The current language might conflate the status of the user with the status of the transaction. KYC requirements can change and different tiers can be invoked in the future so a wallet should make a concise decision wrt what this status means in the context of a user.

Reply:
Yeah, that's a good idea.